### PR TITLE
 #163426521 setup email service 

### PR DESCRIPTION
**Pull request by biolabalo**
#### What does this PR do?
- Send email notification to users

#### Description of Task to be completed?
-  Install  @sendgrid/mail from Sendgrid
- Write Helper Function to send Email

#### What are the relevant pivotal tracker stories?

- [#163426521](https://www.pivotaltracker.com/story/show/163426521)

#### Screen Shots
screen shot 2019-02-05 at 12 00 38 pm

### CODE REVIEW

**Review by omenkish — DISMISSED** [review]

**Review by ozone4real — DISMISSED** [review]
LGTM

**Review by primuse — DISMISSED** [review]
Looks good

**Review by EfeAgare — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -409,3 +409,8 @@ No additional parameters required
 ### Get Tags
 
 `GET /api/tags`
+
+
+<hr>
+
+You can access the app here https://indiana-ah-staging.herokuapp.com/
```
Comment (p-funky) [inline README.md]: @primuse @EfeAgare @omenkish @ozone4real how come we all reviewed this PR and did not notice this is not meant to be here?

**Review by EfeAgare — COMMENTED** [review]
Please Make your JsDoc comment more descriptive - like @param {*} receiverEmail

**Review by EfeAgare — APPROVED** [review]

**Review by p-funky — APPROVED** [review]

**Pull request merged**